test/test_datasets_download.py

```diff
@@ -150,7 +162,7 @@ def assert_server_response_ok():
 
 
 def assert_url_is_accessible(url, timeout=5.0):
-    request = Request(url, headers={"method": "HEAD", "User-Agent": USER_AGENT})
+    request = Request(url, headers={"User-Agent": USER_AGENT}, method="HEAD")
```

Comment:
for my understanding: is this a cosmetic change?

Reply:
I didn't mean to include this. It was just for testing. But it shouldn't matter.

Reply:
Ok, if you want to revert this change in a follow-up PR I'm ok with it